ext/standard/tests/general_functions/error_get_last.phpt

```diff
@@ -15,7 +15,18 @@ $a = $b;
 
 var_dump(error_get_last());
 
-echo "Done\n";
+function trigger_fatal_error_with_stacktrace() {
+    ini_set('fatal_error_backtraces', true);
+
+    eval("class Foo {}; class Foo {}");
+}
+
+register_shutdown_function(function() {
+    var_dump(error_get_last());
+    echo "Done\n";
+});
+
+trigger_fatal_error_with_stacktrace();
```

Comment:
It would probably make sense to split this into a dedicated test file, given it's fatal.